intermediate/01-high-level-computation-patterns.ipynb

```diff
@@ -141,13 +138,26 @@
    },
    "outputs": [],
    "source": [
+    "da: xr.DataArray = xr.tutorial.load_dataset(\"air_temperature\", engine=\"netcdf4\").air\n",
```

Comment:
Need a type hint on da in order for subsequent methods to be picked up (`da.resample`). Also, if chaining methods only the first is picked up `da.isel(time=0).plot()` links to `isel` but not `plot`

Reply:
opened discussion for clarification on this https://github.com/felix-hilden/sphinx-codeautolink/discussions/146